src/app/components/ui/MainListElement/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Typography from 'app/components/ui/Typography';
+import Icon from 'app/components/ui/Icon';
+import ListItem from '@material-ui/core/ListItem';
+import Divider from '@material-ui/core/Divider';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import IconButton from '@material-ui/core/IconButton';
+import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
+
+const MainListElement = (
+  {
+    title,
+    subtitle,
+    subtitle2,
+    icon,
+    actionIcon,
+    action,
+    informative,
+    ...props}
```

Comment:
this bracket should be in the next line 
```suggestion
    ...props
    })
```

Reply:
Done